test/ci-operator-integration/multi-stage/configs/master/openshift-hyperkube-master.yaml

```diff
@@ -38,7 +38,7 @@ tag_specification:
   namespace: ci
 tests:
 - as: multi-stage
-  steps:
+  literal_steps:
```

Comment:
This seems odd -- why? Users should never set this

Reply:
This is part of the `ci-operator`'s integration tests, so this is supposed to be a config that the ci-operator would get from the resolver (or pj-rehearse when we update that to support registry), which would have converted the `steps` to `literal_steps` during resolution. With the other changes made to multistage, this section would simply be ignored if it was just `steps`.

Reply:
Ah, so this is something served by the registry? I was expecting this file to be read from disk in this test if the aim of the test is not to use the resolver.